share/geojson/README.md

```diff
@@ -30,6 +30,10 @@ Mac: `brew install gdal`
     ogr2ogr -f "GeoJSON" -t_srs EPSG:4326 world-boundary.json World_Countries__Generalized_.shp
     ```
 
+### US Boundary
+
+1. Download GeoJSON file [open data](https://public.opendatasoft.com/explore/dataset/us-county-boundaries/export/)
```

Comment:
Thank you for the feedback! I did that in the most recent commit. I made the mapping from abrr to full name inside the `ImportUSBoundary` function. Thinking about moving it elsewhere because it makes the function quite long.